WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,21 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/regulations/wca-regulations.json")).freeze
+  rescue
+    REGULATIONS = [].freeze
```

Comment:
I would rather our server fail to start up than silently ignore this missing file. Otherwise, I'm not sure that we'll notice when this breaks. How about just removing the `begin...rescue` entirely?